scripts/genertate/README.md

```diff
@@ -0,0 +1,33 @@
+
+Some of the examples below may include Unicode text characters. Set the environment variable:
+```bash
+export PYTHONIOENCODING=UTF-8
+```
+
+Use the following command to generate gpt2 samples
+```bash
+python3 generate_unconditional_gpt2_samples.py \
+    --model_name gpt2_774M \
+    --nsamples 5000 > samples
+```
+
+```bash
+python3 interactive_conditional_gpt2_samples.py \
+    --model_name gpt2_774M \
+    --nsamples 1
+```
+
+```bash
+python3 calculate_metrics.py samples
+```
+
+# 贴点sample
+
+Some metrics for the unconditional generated text
```

Comment:
Let's mention https://arxiv.org/pdf/1904.09751.pdf. Also, let's state that these metrics are just heuristics and there is no guarantee that they correlates well with human evaluation.